src/gui/owncloudgui.cpp

```diff
@@ -814,7 +814,7 @@ void ownCloudGui::setupActions()
 {
     _actionStatus = new QAction(tr("Unknown status"), this);
     _actionStatus->setEnabled(false);
-    _actionSettings = new QAction(tr("Settings..."), this);
+    _actionSettings = new QAction(tr("Show %1").arg(Theme::instance()->appNameGUI()), this);
```

Comment:
I suggest to use `Show <app name> settings`, which is what we use in the desktop actions (the context menu entries on Linux desktops).

Reply:
Settings if completely missleading because we display our ui and not the settings

Reply:
You're right. We should possibly change this elsewhere, too. The CLI parameter `-s` stands for settings too, same goes for various context menus, etc. Perhaps we should open an issue for this.